test/strategy/EnrichClientFromRemoteConfigStrategyTest.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+var stormpath = require('stormpath');
+
+var common = require('../common');
+var _ = common._;
+var assert = common.assert;
+var sinon = common.sinon;
+
+var strategy = require('../../lib/strategy');
+var EnrichClientFromRemoteConfigStrategy = strategy.EnrichClientFromRemoteConfigStrategy;
+
+describe('EnrichClientFromRemoteConfigStrategy', function () {
+  var client, mockApplications, testStrategy;
+
+  before(function (done) {
+    mockApplications = [{
+      href: 'https://stormpath.mock/applications/stormpath',
+      name: 'Stormpath'
+    }, {
+      href: 'https://stormpath.mock/applications/my-application',
+      name: 'My Application'
+    }];
+
+    client = new stormpath.Client({
+      skipRemoteConfig: true
+    });
+
+    testStrategy = new EnrichClientFromRemoteConfigStrategy(function () {
+      return client;
+    });
+
+    client.on('error', function (err) {
+      throw err;
+    });
+
+    client.on('ready', function () {
+      done();
+    });
+  });
+
+  it("should resolve application by name", function (done) {
```

Comment:
It's being used inconsistently throughout the project. So what I'm saying is just that it's not consistent but that we should use ESLint so that it is. But I'll fix this, this time.